### PR TITLE
Add nginx-vts-exporter

### PR DESCRIPTION
**Pull request by hnlq715**
@juliusv @mpalmer @alex

### CODE REVIEW

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Apache exporter](https://github.com/neezgee/apache_exporter)
    * [HAProxy exporter](https://github.com/prometheus/haproxy_exporter) (**official**)
    * [Nginx metric library](https://github.com/knyar/nginx-lua-prometheus)
+   * [Nginx vts exporter](https://github.com/hnlq715/nginx-vts-exporter)
```
Comment (beorn7) [inline content/docs/instrumenting/exporters.md]: Capitalize acronyms: VTS

**Comment by beorn7** [comment]
Thanks.

Note that you are not following https://prometheus.io/docs/instrumenting/writing_exporters/#collectors , i.e. you should use `NewConstMetric` to create metrics on the fly rather then setting and resetting them all in each scrape cycle. This will also allow you to use counters (rather than gauges) where appropriate.

(We can still link the exporter from the Prometheus site, so no need to do it now. But please consider it for later.)

**Comment by hnlq715** [comment]
@beorn7 Thanks. I'll consider the point you figured out.

**Comment by hnlq715** [comment]
I've refactored nginx-vts-exporter with NewConstMetric to create metrics 😄

**Pull request merged**